scripts/package-harvester-os

```diff
@@ -74,3 +74,19 @@ xorriso -osirrox on -indev ${ARTIFACTS_DIR}/${ISO_PREFIX}.iso -extract /rootfs.s
 cd ${ARTIFACTS_DIR}
 CHECKSUM_FILE=${ISO_PREFIX}.sha512
 sha512sum ${PROJECT_PREFIX}* > $CHECKSUM_FILE
+
+# Write version.yaml
+if [[ -v DRONE_TAG ]]; then
```

Comment:
Would `-n` be more suitable? `-v` succeeds even when the variable is an empty value.

```suggestion
if [[ -n "${DRONE_TAG}" ]]; then
```